src/blockdata/witness.rs

```diff
@@ -43,7 +43,10 @@ pub struct Witness {
 }
 
 /// Support structure to allow efficient and convenient iteration over the Witness elements
-pub struct Iter<'a>(::core::slice::Iter<'a, u8>);
+pub struct Iter<'a> {
+    inner: ::core::slice::Iter<'a, u8>,
```

Comment:
nit: We don't need these leading colons anymore, since the MSRV bump.

Reply:
fixed